src/components/StudyInfo/CavaticaCounts.js

```diff
@@ -65,4 +85,4 @@ CavaticaCounts.defaultProps = {
   projects: [],
 };
 
-export default CavaticaCounts;
+export default withAnalyticsTracking(CavaticaCounts, {logToConsole: true});
```

Comment:
Why do you need this? Doesn't the amplitude provider pass down all the inherited info you need?

Reply:
because I think render props are messy and this allows you to add `eventProperties` prop directly to the component. It also infuses the renderProps in AmplitudeProxy with the EVENT_CONSTANTS and allows use to hook into other dev. exp. conveniences like logging events to the console and saving event json schemas to a local file when building out new new events